_basics/truthy-and-falsy.md

```diff
@@ -3,47 +3,43 @@ title: Truthy and falsy
 description: An overview of boolean logic in the Liquid template language.
 ---
 
-In programming, anything that returns `true` in a conditional is called **truthy**. Anything that returns `false` in a conditional is called **falsy**. All object types can be described as either truthy or falsy.
-
-- [Truthy](#truthy)
-- [Falsy](#falsy)
-- [Summary](#summary)
+When a non-boolean [data type]({{ "/basics/types/" | prepend: site.baseurl }}) is used in a boolean context (such as a conditional tag), Liquid decides whether to evaluate it as `true` or `false`. Data types that return `true` by default are called **truthy**. Data types that return false by default are called **falsy**.
 
 ## Truthy
 
 All values in Liquid are truthy except `nil` and `false`.
```

Comment:
Which paragraph did you have in mind?

Reply:
Ah yes, realize this might be ambiguous from GitHub's highlighting. What do you think of sticking `All values in Liquid are truthy except nil and false.` right on the end of the highlighted paragraph? We'd get:

> When a non-boolean [data type]({{ "/basics/types/" | prepend: site.baseurl }}) is used in a boolean context (such as a conditional tag), Liquid decides whether to evaluate it as `true` or `false`. Data types that return `true` by default are called **truthy**. Data types that return false by default are called **falsy**. All values in Liquid are truthy except `nil` and `false`.

This suggestion is kind of outside the scope of your changes, so we can ignore it if you prefer.

Reply:
I think it's more visible as its own standalone paragraph than as the last sentence of a larger paragraph.